pom.xml

```diff
@@ -71,7 +71,8 @@
     </dependencyManagement>
 
     <modules>
-	<module>adapter</module>
+	      <module>adapter</module>
```

Comment:
Different indentation :)

Reply:
@martin-kanis I believe the current indentation is broken. Take a look at https://github.com/redhat-developer/redhat-sso-boms/blob/7.3.x/pom.xml#L74